### ISSUE TITLE
Getting error while importing axios-curlirize on ts + node

### ISSUE THREAD

**Issue by lambainsaan** [delirius325/axios-curlirize#43]
Getting error while importing axios-curlirize,

ts-node ver. 9.1.1, typescript ver. 4.2.4

```
[ERROR] 14:27:26 Error: Must use import to load ES Module: node_modules/axios-curlirize/src/main.js
require() of ES modules is not supported.
require() of node_modules/axios-curlirize/src/main.js from temp.ts is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename main.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from node_modules/axios-curlirize/package.json.
```

**Comment by lambainsaan** [comment]
Importing the package using

```js
import curlirize from 'axios-curlirize';
console.log(curlirize)
```

**Comment by youngcm2** [comment]
Same issue

**Comment by delirius325** [comment]
I believe this is due to an older version of Node; `axios-curlirize` now makes use of native Node modules. I see two potential options here:

* Upgrade your version of Node to a more recent version (on CircleCI, we're using 16.4)
* Create a PR to fix the issue (I'll gladly review it) because sadly I don't have the time to fix it at this moment.

Thank you!

**Comment by thib3113** [comment]
Same issue here . 

@delirius325 pls use lts insteed of last node version on circle CI . last LTS is 14.17.1

**Comment by delirius325** [comment]
Aside from using the new native Node modules and linting fixes, there's nothing new in `1.4.0`. Please use `1.3.7` if you're experiencing this error.

**Comment by ShayBox** [comment]
This needed to be a major version change, switching to ES modules is a major breaking change, now everyone and every package depending on axios-curlirize 1.x.x^ will automatically use a broken update.
